python/mypy.ini

```diff
@@ -41,9 +41,15 @@ disallow_untyped_defs = False
 [mypy-pyspark.join]
 disallow_untyped_defs = False
 
+[mypy-pyspark.ml.*]
+disallow_untyped_defs = False
+
```

Comment:
Let's not do that.

Reply:
Okie doke. I was trying to stick to the annotations that were already in the `.pyi` files, but I will go ahead an add the missing return types to all functions.

Reply:
Thanks!